src/Makefile

```diff
@@ -171,13 +176,15 @@ endif
 
 install-strip: install
 	strip -s $(bindir)/kak
+	strip -s $(bindir)/kak-setsid
 
 uninstall:
 	rm -rf $(bindir)/kak \
+		$(bindir)/kak-setsid
 		$(sharedir) \
 		$(docdir) \
 		$(mandir)/kak.1.gz \
 		$(mandir)/kak.1
 
 .PHONY: check TAGS clean dist distclean installdirs install install-strip uninstall
-.PHONY: tags test man kak FORCE
+.PHONY: tags test man kak kak-setsid FORCE
```

Comment:
The `kak-setsid` is an actual file-target, so it probably shouldn't be declared as being phony.